.env

```diff
@@ -1,5 +1,5 @@
 # Configuration for the LLM service
-LLM_BASE_URL=http://model-runner.docker.internal/engines/llama.cpp/v1
+LLM_BASE_URL=http://llm:8080/v1
```

Comment:
This file should be removed, since the new Compose integration injects these URLs.